### DISCUSSION TITLE
Is development stopped for this framework ?

### DISCUSSION THREAD

**Discussion by Mouli04** [vuejs/petite-vue#225] [General]
Is development stopped for this framework ? 
since, no changes in the code.

**Comment by sqllyw** [comment]
somebody mentioned , it is not abandoned but 'done'

https://github.com/vuejs/petite-vue/discussions/53#discussioncomment-4483647

**Reply by PetesBreenCoding** [reply]
Surely no software is ever "done"? There are always at least bug fixes, performance improvements and dependency updates

**Reply by sqllyw** [reply]
good points

**Reply by jikkuatwork** [reply]
Perhaps we can create a community and revive it: https://github.com/vuejs/petite-vue/discussions/235
